docs/openstack-quota.md

```diff
@@ -7,7 +7,7 @@ To read more about Openstack quotas please visit the [upstream docs](https://doc
 You can view your quotas via Skyline or using the OpenStack CLI client.  The Skyline home page is the easiest way to view your quota limits and utilization in one place.
 ![SkylineHome](./assets/images/SkylineHomeQuota.png)
 
-You can also view quota information using the OpenStack CLI, however, you'll need to use a couple of commands.  
+You can also view quota information using the OpenStack CLI, however, you'll need to use a couple of commands.<BR>

```

Comment:
Not sure if we want to add such chars, mostly for consistency sake. 

You can run the pre-commit stuff that also happens during the pr checks and it'll resolve those issues for you. 
re; pre-commit run --show-diff-on-failure --color=always --files docs/<myfile> docs/<myotherfile>

Reply:
@the2hill the pre-commit checks were failing when using two spaces for a line break, so I tried <BR>, which is apparently also valid markdown, that resolved the pre-commit checks, worked in mkdocs, and looked good in the VSCode preview.  What's the preferred forced line break?

Reply:
apparently "<BR>" works in here too since it supports markdown